### DISCUSSION TITLE
`-etags` issue!

### DISCUSSION THREAD

**Discussion by 0xhunster** [projectdiscovery/nuclei#4897] [Q&A] [answered]
Command: `nuclei -target "https://hackerone.com" -stats -duc -etags "ssl,fuzzing" -t ~/nuclei-templates/http/ -silent`
Nuclei version: `v3.1.10`
Installation method: go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
Golang version: `go1.22.0`
Platform: linux/amd64

when I used the normal command without `-etags`, I saw templates loaded `Templates: 7060`.
But when I used the command `-etags "ssl,fuzzing"`, I saw the same templates loaded `Templates: 7060`. These templates would be excluded.

![image](https://github.com/projectdiscovery/nuclei/assets/46501627/9d024561-49db-4e8d-be10-18359802d99c)

**Comment by olearycrew** [comment, marked as answer]
@0xhunster this would be expected because of your use of `-t ~/nuclei-templates/http/`

* All of the `ssl` tagged templates are in the `ssl` folder
* Fuzzing templates are excluded by default unless you include them (see [this docs page](https://docs.projectdiscovery.io/tools/nuclei/running#executing-nuclei-templates) for the default exclusion details).

Thus, you are already filtered down to the templates in the `http` folder, and the `etags` doesn't filter anymore out. I confirmed this with these commands:

```bash
nuclei -u https://boleary.dev -stats -duc -t http/ -silent
nuclei -u https://boleary.dev -stats -duc -t http/ -silent -etags ssl,fuzzing
```

vs

```bash
nuclei -u https://boleary.dev -stats -duc -silent
nuclei -u https://boleary.dev -stats -duc -silent -etags ssl,fuzzing

```
